app/api/toHtml/route.ts

```diff
@@ -32,7 +42,7 @@ export async function POST(request: Request) {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
-        Authorization: `Bearer ${process.env.OPENAI_API_KEY}`,
+        Authorization: `Bearer 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`,
```

Comment:
Hope you delete this API key

Reply:
Probably already out $120 if he's on tier 1 and didn't set a soft limit.

Reply:
GOODNESS!!! Thanks for spotting that. I checked, and fortunately, I have already revoked the key for some reason.

Reply:
Happens to the best of us at least once. Might still want to set a soft limit anyway, that's saved my wallet from a skynet situation more than once.

Reply:
Yeah, that's something I should do really.